.github/workflows/build.yaml

```
@@ -0,0 +1,32 @@
+---
+name: Test container builds
+
+"on":
+  pull_request:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Code checkout
+        uses: actions/checkout@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64
+          push: false
+          build-args: |
+            RUBY_VERSION: 3.0.0
+            TERRAFORM_VERSION: 1.2.4
+          tags: |
+            ghcr.io/${{ github.repository }}:edge
```

I think this might need to be
```suggestion
            ghcr.io/${{ github.repository }}:${TERRAFORM_VERSION}-edge
```
You may want to add this when adding the `matrix` though